schema/me/artwork_inquiry_requests.js

```diff
@@ -0,0 +1,79 @@
+import gravity from '../../lib/loaders/gravity';
+import date from '../fields/date'
+import Artwork from '../artwork/index';
+
+import {
+  GraphQLList,
+  GraphQLInt,
+  GraphQLBoolean,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLID,
+  GraphQLEnumType
+} from 'graphql';
+
+export const ArtworkInquiryRequestType = new GraphQLObjectType({
+  name: 'ArtworkInquiryRequest',
+  fields: () => ({
+    id: {
+      type: GraphQLID,
+    },
+    inquirer: {
+      type: new GraphQLObjectType({
+        name: 'ArtworkInquiryRequestInquirer',
+        fields: () => ({
+          id: { type: GraphQLID },
+          name: { type: GraphQLString, }
+        }),
+      })
+    },
+    artwork: {
+      type: Artwork.type,
+      resolve: ({ inquireable }) => inquireable,
+    },
+    statuses: {
+      type: new GraphQLList(
+        new GraphQLObjectType({
+          name: 'ArtworkInquiryRequestStatus',
+          fields: () => ({
+            id: { type: GraphQLID },
+            title: { type: GraphQLString },
+            note: { type: GraphQLString },
+            created_at: date
+          }),
+        })
+      ),
+    },
+    status: {
+      type: GraphQLInt,
+    },
+    user_reported_outcome: {
+      type: new GraphQLEnumType({
+        name: 'UserReportedOutcome',
+        values: {
+          PURCHASED: { value: 'purchased' },
+          STILL_CONSIDERING: { value: 'still_considering' },
+          HIGH_PRICE: { value: 'high_price' },
+          LOST_INTEREST: { value: 'lost_interest' },
+          WORK_UNAVAILABLE: { value: 'work_unavailable' },
+          OTHER: { value: 'other' },
+        },
+      }),
+    },
+    created_at: date,
+  }),
+});
+
+export default {
+  type: new GraphQLList(ArtworkInquiryRequestType),
+  description: 'A list of the current user’s inquiry requests',
+  args: {
+    artwork_id: { type: GraphQLID },
+  },
+  resolve: (root, options, request, { rootValue: { accessToken } }) => {
+    if (!accessToken) return null;
+    return gravity.with(accessToken)('me/artwork_inquiry_requests',
+      { inquireable_type: 'artwork' }
```

Comment:
Ok, https://github.com/artsy/gravity/pull/10826 is deployed so we can switch this to just use `me/inquiry_requests` (with the same args).